Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.cs

```diff
@@ -81,5 +106,150 @@ protected override void OnApplyTemplate()
 
             base.OnApplyTemplate();
         }
+
+        private void OnApplyTemplateNavView()
+        {
+            if (_navigationView is NavigationView navView && navView != null)
+            {
+                navView.ItemInvoked -= NavigationViewItemInvoked;
+                navView.SelectionChanged -= NavigationViewSelectionChanged;
+                navView.Loaded -= NavigationViewLoaded;
+            }
+
+            navView = (NavigationView)GetTemplateChild("NavView");
```

Comment:
If someone override the template NavView wouldn't exist, I think it is better to use as and return if null

Reply:
Done